### ISSUE TITLE
About masked face recognition

### ISSUE THREAD

**Issue by SongJ123** [mk-minchul/AdaFace#27]
Thank you for your excellent work! I tried your pretrained model on 1k-face recognition and it just worked very well!
Now I'm trying masked face recognition. I augmented the dataset faces_emore by putting 5 kinds of masks on the face, so the number of training imgs is 5x bigger than the original dataset. I'm wondering how many epochs I should set if I want to get a good 'ir50' model? (or maybe 'ir18' model). Do you have any suggestions on training this model? I intend to use the same parameters you show in the readme. Since the training process could take a long time, it would be very helpful if you can give me some advice to avoid several trainings.

**Comment by SongJ123** [comment]
Thank you, I will try.

**Comment by SongJ123** [comment]
> Hi SongJ123, I think the default setting could serve as a good starting point! Roughly speaking from my own experience, as the dataset becomes larger, you have more steps during optimization, so there is no need to increase the epochs. But if the task becomes difficult, then it may need more epochs or a bigger model.

By the way, can you tell me the training parameters you used to train the pretrained 'ir_50' model, which is provided in the model zoo(R50 trained by MS1MV2), for reference.

**Comment by mk-minchul** [comment]
The setting I used is 
```
python main.py \
    --gpus 2 \
    --use_16bit \
    --batch_size 512 \
    --num_workers 16 \
    --epochs 26 \
    --lr_milestones 12,20,24 \
    --arch ir_50 \
    --lr 0.1 \
    --head adaface \
    --m 0.4 \
    --h 0.333 \
    --low_res_augmentation_prob 0.2 \
    --crop_augmentation_prob 0.2 \
    --photometric_augmentation_prob 0.2
```
